Dockerfile

```diff
@@ -301,18 +301,13 @@ FROM grass_gis as grass_gis_final
 
 # GRASS GIS specific
 # allow work with MAPSETs that are not owned by current user
-# add GRASS GIS envs for python usage
-ENV GRASSBIN="/usr/local/bin/grass" \
-    GRASS_SKIP_MAPSET_OWNER_CHECK=1 \
+ENV GRASS_SKIP_MAPSET_OWNER_CHECK=1 \
     SHELL="/bin/bash" \
```

Comment:
If the shell wanted to be used inside the image is bash, wouldn't it need to be set with `SHELL` instead? https://docs.docker.com/reference/dockerfile/#shell

Otherwise, the default shell (and also the one used for RUN, CMD, and ENTRYPOINT) would be `sh`, and we probably should make sure any shell script intended to be run would be compatible with sh (not rely on bash-isms)